src/ansys/systemcoupling/core/participant/manager.py

```diff
@@ -140,6 +140,9 @@ def solve(self):
         if self.__solve_exception:
             raise self.__solve_exception
 
+    def upload_file(self, file_name: str):
+        self.__syc_session.upload_file(file_name)
+
```

Comment:
In an active PR ([PR 312](https://github.com/ansys/pysystem-coupling/pull/312)), I have come to the conclusion that it is better to pass in the `Session` object to `get_injected_command_map`, rather than just the root API object. The above should be OK for now, but I expect it to be possible to remove it soon.

Reply:
Thanks for letting me know.